rasa/nlu/selectors/response_selector.py

```diff
@@ -204,6 +210,9 @@ def required_components(cls) -> List[Type[Component]]:
         MASKED_LM: False,
         # Name of the intent for which this response selector is to be trained
         RETRIEVAL_INTENT: None,
+        # Boolean flag to check if actual text of the response
+        # should be used as ground truth label for training the model.
+        USE_TEXT_AS_LABEL: True,
```

Comment:
Reminder for myself: Change this to False before merging. This is `True` for model running model regression tests once.